Sources/Fuzzilli/Protobuf/ProtoUtils.swift

```diff
@@ -64,3 +66,57 @@ class OperationCache {
         operations.append(operation)
     }
 }
+
+public func encodeProtobufCorpus<T: Collection>(_ programs: T) -> (Int, Data) where T.Element == Program {
```

Comment:
Actually, thinking more about this, I think this function should just throw when an encoding error occurs, like `decodeProtobufCorpus` does. Then the whole `do {} catch` block isn't necessary and the return type again becomes just `Data`. WDYT?

Reply:
Looking at my current error log for running the compiler against a large JS file corpus, all current errors are caught here: https://github.com/googleprojectzero/fuzzilli/pull/113/files#diff-9492c4d06ed1e3e95cd0b62407fba428R129, and I don't see expect many errors popping up going from a list of Fuzzilli Programs to the final corpus file. So, we should be good here!